jib-maven-plugin/pom.xml

```diff
@@ -311,6 +335,7 @@
         <configuration>
           <excludes>
             <exclude>**/*IntegrationTest.java</exclude>
+            <exclude>**/*IT.java</exclude>
```

Comment:
This pattern violates Google coding conventions (camel case and no abbreviations) and shouldn't be occurring in our repos.